Mage.Sets/src/mage/sets/JuniorApacSeries.java

```diff
@@ -0,0 +1,30 @@
+package mage.sets;
+
+import mage.cards.ExpansionSet;
+import mage.constants.Rarity;
+import mage.constants.SetType;
+
+/**
+ * https://scryfall.com/sets/pjas
+ */
+public class JuniorApacSeries extends ExpansionSet {
```

Comment:
Needs to be `JuniorAPACSeries`. Already corrected locally and will be fixed on next push.